tests/components/geo_location/test_usgs_earthquake_hazards_program_feed.py

```diff
@@ -0,0 +1,208 @@
+"""The tests for the USGS Earthquake Hazards Program Feed platform."""
+import datetime
+import unittest
+from unittest import mock
+from unittest.mock import patch, MagicMock
+
+from homeassistant.components import geo_location
+from homeassistant.components.geo_location import ATTR_SOURCE
+from homeassistant.components.geo_location\
+    .usgs_earthquake_hazards_program_feed import \
+    ATTR_ALERT, ATTR_EXTERNAL_ID, SCAN_INTERVAL, ATTR_PLACE, \
+    ATTR_MAGNITUDE, ATTR_STATUS, ATTR_TYPE, \
+    ATTR_TIME, ATTR_UPDATED, CONF_FEED_TYPE
+from homeassistant.const import EVENT_HOMEASSISTANT_START, \
+    CONF_RADIUS, ATTR_LATITUDE, ATTR_LONGITUDE, ATTR_FRIENDLY_NAME, \
+    ATTR_UNIT_OF_MEASUREMENT, ATTR_ATTRIBUTION, CONF_LATITUDE, CONF_LONGITUDE
+from homeassistant.setup import setup_component
+from tests.common import get_test_home_assistant, assert_setup_component, \
+    fire_time_changed
+import homeassistant.util.dt as dt_util
+
+CONFIG = {
+    geo_location.DOMAIN: [
+        {
+            'platform': 'usgs_earthquake_hazards_program_feed',
+            CONF_FEED_TYPE: 'past_hour_m25_earthquakes',
+            CONF_RADIUS: 200
+        }
+    ]
+}
+
+CONFIG_WITH_CUSTOM_LOCATION = {
+    geo_location.DOMAIN: [
+        {
+            'platform': 'usgs_earthquake_hazards_program_feed',
+            CONF_FEED_TYPE: 'past_hour_m25_earthquakes',
+            CONF_RADIUS: 200,
+            CONF_LATITUDE: 15.1,
+            CONF_LONGITUDE: 25.2
+        }
+    ]
+}
+
+
+class TestUsgsEarthquakeHazardsProgramFeedPlatform(unittest.TestCase):
```

Comment:
We now prefer that new tests are standalone pytest test functions. See eg the deconz test suite.

Reply:
OK